src/mainwindow.cpp

```diff
@@ -682,13 +682,13 @@ void MainWindow::askDeleteFile()
 
     if (!fileInfo.isWritable())
     {
-        QMessageBox::critical(this, tr("Error"), tr("Can't delete %1:\nNo write permission or file is read-only.").arg(fileName));
+        QMessageBox::critical(this, tr("Error"), tr("Could not delete %1:\nGrant write permission or ensure the file is not read-only.").arg(fileName));

```

Comment:
The idea is to describe the action needed to resolve the problem, not have the user figure out what to do on account of having the problem described.

Reply:
That makes sense, but I think "Grant write permission" is a bit aggressive, since you typically won't want to respond to this by granting write permissions. Maybe "Ensure that file has write permission and is not read-only.